Source/WebCore/platform/gstreamer/PlatformSpeechSynthesizerGStreamer.cpp

```diff
@@ -136,6 +149,16 @@ void GstSpeechSynthesisWrapper::speakUtterance(RefPtr<PlatformSpeechSynthesisUtt
 
     m_utterance = WTFMove(utterance);
     webKitFliteSrcSetUtterance(WEBKIT_FLITE_SRC(m_src.get()), m_utterance->voice(), m_utterance->text());
+
+    // The pitch element does not handle the rate bigger than 1.0. We control
```

Comment:
`m_pitchElement` might be null here, so needs a check :)

Reply:
Done.